actioncable/lib/action_cable/connection/subscriptions.rb

```diff
@@ -42,6 +42,8 @@ def remove(data)
 
       def remove_subscription(subscription)
         subscription.unsubscribe_from_channel
+      ensure
```

Comment:
Nah, `unsubscribe_from_channel` is just user callbacks all the way down far as I can tell. This seems right.